examples/README.md

```diff
@@ -1,43 +1,15 @@
 # Yew Examples
 
-In order to run the examples, we provide the `run_example.sh` script.
-This script takes care of compiling the example and runs a web server for you.
-All that's left is for you to play around with the examples :).
-
-> **A note for Windows users:**<br>
-> Depending on how you installed `git` you will already have a bash emulator at your disposal. This allows you to run the `run_example.sh` script (and any other bash script) normally.<br>
-> See <https://gitforwindows.org/#bash> for more information.
->
-> We're always trying to improve the developer experience for developers across all platforms.
-> There's an ongoing effort to replace the bash scripts with a Rust command line tool ([#1418](https://github.com/yewstack/yew/issues/1418)).
-> If at any point you encounter an issue, don't hesitate to ask a question or open an issue.
-
 ## Dependencies
 
-Before we can run the examples we need to get a few things ready.
-
-Some examples currently use `wasm-bindgen` and others use `wasm-pack`.
-You can install them both using the following command:
+The examples are built with [trunk](https://github.com/thedodd/trunk).
+You can install it with the following command:
 
 ```bash
-cargo install wasm-pack wasm-bindgen-cli
+# at some point in the future, trunk should automatically download wasm-bindgen for you
+cargo install trunk wasm-bindgen-cli
```

Comment:
Just as a placeholder note, with the upcoming `0.4.0` release, binaries for Linux, MacOS & Windows will all be available as part of every release. I'm hoping to have that all finished up tonight.

Reply:
Of course, for now that will only cover trunk, not wasm-bindgen-cli ... which I'm planning on knocking out in the future.

Reply:
Ah, that will be great for CI too!

Reply:
Is the plan for `wasm-bindgen-cli` to copy `wasm-pack` or do you have something else in mind?

Reply:
We've got some good discussion going on here: https://github.com/thedodd/trunk/issues/5 I need to sync up with Ashley again on that soon. I might have to jump into the `wasm-pack` code and pull those bits out. We'll see. It would be SO nice not to have to worry about installing and keeping `wasm-bindgen-cli` in sync with the version of `wasm-bindgen` being used in the `Cargo.toml`, which is what `wasm-pack` is already doing.